feat: Add a copy URI icon

HagerDakroury opened the pull request:
### Checklist

- [x] I indicated which issue (if any) is closed with this PR using [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] I only changed lines related to my PR (no bulk reformatting)
- [x] I indicated the source and check the license if I re-use code, or I did not re-use code
- [x] I made my best to solve only one issue in this PR, or explain why multi had to be solved at once.

### Issue

closes #40 

### Details

![copyURI](https://user-images.githubusercontent.com/37930475/120689012-ee338800-c4a3-11eb-90d3-1f909c283d6e.gif)

### Further Enhancements
1. Make the icon's color match the URI
2. Fix the tooltip position (looks like a common issue,see #210 , and according to a quick search will need either overriding the bootstrap's tooltip or creating a one using pure CSS or a third option)

bryan-brancotte reviewed (changes requested):
Hi @HagerDakroury 

Great for tackling the issue, @matuskalas who will be happy to have a less un-intuitive functionality. I have remarks considering the look-and-feel, and also I want to extends this copy ability to all terms. I am adding it to the issue

HagerDakroury commented:
> Hi @HagerDakroury
> 
> Great for tackling the issue, @matuskalas who will be happy to have a less un-intuitive functionality. I have remarks considering the look-and-feel, and also I want to extends this copy ability to all terms. I am adding it to the issue

Converting this to a draft and moving the discussion to the issue at this moment 👍 And love to hear what @matuskalas thinks concerning the design too!

HagerDakroury commented:
This is how it looks like now after adapting the design you suggested @bryan-brancotte . Would that be good to go with a clearer indication that it's a copy button?

![cpyNew](https://user-images.githubusercontent.com/37930475/121011944-7e6d1800-c797-11eb-8979-9ba63f8d2f45.gif)

bryan-brancotte reviewed (commented):
Hi @HagerDakroury 
Super great, and I was about to say something about the html ID but you also already fixed it !
A feature request : when we click to copy an element, maybe the others copy-buttons who went from `fa-check to `fa-check` should go back to `fa-copy`.

HagerDakroury commented:
> Hi @HagerDakroury
> Super great, and I was about to say something about the html ID but you also already fixed it !
> A feature request : when we click to copy an element, maybe the others copy-buttons who went from `fa-check to `fa-check`should go back to`fa-copy`.

Yeah, I actually was thinking about how to tackle this.

Do you think it should be a timeout thing? (like Github, the icon is toggled back after ~3 seconds) or whenever a new button is clicked?

bryan-brancotte commented:
I did not though of the timeout things, but it is way better than what I proposed initially.

HagerDakroury commented:
> I did not though of the timeout things, but it is way better than what I proposed initially.

Added. With a timeout of 1 sec (the Copied! tooltip is also removed after the timeout)

![cpyTimeout](https://user-images.githubusercontent.com/37930475/121022936-17556080-c7a3-11eb-9804-a1d3cbf64abc.gif)

bryan-brancotte commented on css/index.css:
```diff
@@ -248,4 +248,15 @@ svg{
     user-select:none;
 }
+
+.cpy-clipboard {
```
Dead code ?

HagerDakroury replied:
Done

bryan-brancotte reviewed (approved):
Perfect, let's squash and merge (once the possible dead code is removed/not dead code)

bryan-brancotte commented:
Squash and merge whenever you fell ready for it :) :trophy:

raashika03 commented on js/tree-edam-stand-alone.js:
```diff
@@ -766,3 +772,27 @@ function toggleFullscreen(){
+    const element = event.srcElement;
+
+    //showing a tooltip indicating the value is copied
+    $(element).attr('title', "URI copied!")
```
Fields in the parents are also getting covered no? So why to use URI specifically?

raashika03 replied:
OK sorry ig I misinterpreted.

HagerDakroury replied:
The URI was added to indicate that a link is copied (not the text). So, even parents are copied by the URI (not their text). What that your question or did I misunderstand? 😃

raashika03 replied:
It should be URL

HagerDakroury replied:
Umm URI was indicated in the issue so we followed that. I think both are technically right.
 What do you think is the most familiar to users? @bryan-brancotte

bryan-brancotte replied:
In the case here it is definitely an [Uniform Resource Identifier](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) , but still thanks for raising the point

raashika03 replied:
`http://edamontology.org/topic_0091`
We're copying `uniform resource locator` specifically 😅 @bryan-brancotte

raashika03 replied:
But it's OK to leave them as uri as URL comes under uri
